From ebee83c277eb5b926d8ff465f61660a6c209fafd Mon Sep 17 00:00:00 2001
From: Edson Juliano Drosdeck <edson.drosdeck@gmail.com>
Date: Thu, 8 Dec 2022 16:51:09 -0300
Subject: [PATCH] hwdb: add Positivo-vaio Pro PW key toggle touchpad (#25669)

(cherry picked from commit 1feb9743d05eadb373caa9aba591871a7d05a278)

Related: RHEL-5950
---
 hwdb.d/60-keyboard.hwdb | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/hwdb.d/60-keyboard.hwdb b/hwdb.d/60-keyboard.hwdb
index 9707132bee..2fee587ed4 100644
--- a/hwdb.d/60-keyboard.hwdb
+++ b/hwdb.d/60-keyboard.hwdb
@@ -2011,6 +2011,8 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnViewSonic:pnVPAD10:*
 ###########################################################
 # Positivo-Vaio
 ###########################################################
+# Vaio Pro (VJPW11F11X)
+evdev:name:AT Translated Set 2 keyboard:dmi:bvn*:bvr*:bd*:svnPositivoBahia-VAIO:pnVJPW11F11X*:pvr*:*
 # Vaio FE14 (VJFE41F11X, VJE42F11X, VJFE44F11X, VJFE54F11X)
 evdev:name:AT Translated Set 2 keyboard:dmi:bvn*:bvr*:bd*:svnPositivoBahia-VAIO:pnVJFE*:pvr*:*
  KEYBOARD_KEY_76=f21		                        # Fn+F1 toggle touchpad
